app/src/main/java/com/example/concordia_campus_guide/Global/SelectingToFromState.java

```diff
@@ -0,0 +1,77 @@
+package com.example.concordia_campus_guide.Global;
+
+import android.location.Location;
+
+import com.example.concordia_campus_guide.Models.Place;
+
+public class SelectingToFromState {
+
+    private static Place from;
+    private static Place to;
+    private static Location myCurrentLocation;
+
+    private static boolean QuickSelect = true;
+    private static boolean SelectTo = false;
+    private static boolean SelectFrom = false;
+
+    public static Place getFrom() {
+        return from;
+    }
+
+    public static void setFrom(Place from) {
+        SelectingToFromState.from = from;
+    }
+
+    public static Place getTo() {
+        return to;
+    }
+
+    public static void setTo(Place to) {
+        SelectingToFromState.to = to;
+    }
+
+    public static Location getMyCurrentLocation() {
+        return myCurrentLocation;
+    }
+
+    public static void setMyCurrentLocation(Location myCurrentLocation) {
+        SelectingToFromState.myCurrentLocation = myCurrentLocation;
+    }
+
+    public static boolean isQuickSelect() {
+        return QuickSelect;
+    }
+
+    public static void setQuickSelectToTrue() {
+        SelectFrom = false;
+        SelectTo = false;
+        QuickSelect = true;
+    }
+
+    public static boolean isSelectTo() {
+        return SelectTo;
+    }
+
+    public static void setSelectToToTrue() {
+        QuickSelect = false;
+        SelectFrom = false;
+        SelectTo = true;
+    }
```

Comment:
what is the difference between this method and  setQuickSelectoTrue() ?? Because the context is the same for both.

Reply:
quick select is when we select for the first time from the first search, select to is when, from the routes page, we decide to change the starting place

Reply:
Can't we use one of the other methods which had the same content ? Because that's what took my attention. The fact that the content of the methods were the same .

Reply:
ok, I see now thank you for clarification  👍